ksqldb-tools/src/test/java/io/confluent/ksql/tools/migrations/util/CommandParserTest.java

```diff
@@ -258,6 +258,28 @@ public void shouldParseTerminateStatement() {
     assertThat(commands.get(0).getCommand(), is("terminate some_query_id;"));
   }
 
+  @Test
+  public void shouldParsePauseStatement() {
```

Comment:
If we're planning to support these statements in the migration tool (seems reasonable to me), we need better test coverage here. At minimum, MigrationsTest.java (the integration test for the migrations tool) needs to be updated.